sky/authentication.py

```diff
@@ -589,6 +593,39 @@ def setup_hyperbolic_authentication(config: Dict[str, Any]) -> Dict[str, Any]:
     # Set up auth section for Ray template
     config.setdefault('auth', {})
     config['auth']['ssh_user'] = 'ubuntu'
-    config['auth']['ssh_public_key'] = public_key_path
```

Comment:
why removing this?

Reply:
def a mistake here since its not related to prime intellect. reverted.